e2e/__tests__/runProgrammaticallyMultipleProjects.test.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {resolve} from 'path';
+import {run} from '../Utils';
+
+const dir = resolve(__dirname, '..', 'run-programmatically-multiple-projects');
+
+test('run programmatically with multiple projects', () => {
+  const {stdout, exitCode} = run(`node run-jest.js `, dir);
+  expect(exitCode).toEqual(0);
+  expect(stdout).toMatch(/Done/);
```

Comment:
could you pass `stdout` through `extractSummaries` or something like that and snapshot it to ensure both tests show up?

Reply:
A great addition, will do that and use snapshotting 👍